server/routes/reporting/standard-report/standard-report.controller.js

```diff
@@ -0,0 +1,206 @@
+const { snakeToCamel } = require('../../../lib/mod-utils');
```

Comment:
this could go inside the iife and be together with the others defs

Reply:
Fixed